### PR TITLE
Add planned_remediation_version and effort_for_fixing fields

### PR DESCRIPTION
**Pull request by ptrovatelli**
**Description**

Add 2 fields: 
- planned remediation version: in what version of the product we plan to remediate this finding
- effort for fixing : is it easy, medium or hard to fix. Help prioritize fixes

**Test results**
[tests.pdf](https://github.com/DefectDojo/django-DefectDojo/files/11019936/tests.pdf)

### CODE REVIEW

File: dojo/templates/dojo/findings_list_snippet.html [hunk 1]
```diff
@@ -94,6 +94,10 @@ <h3 class="has-filters">
                                 <input id="id_bulk_date" name="date" type="date" alt="Select the date the Findings were discovered"/><br/>
                             <label style="display: block" for="id_bulk_planned_remediation_date">Planned Remediation Date</label>
                                 <input id="id_bulk_planned_remediation_date" name="planned_remediation_date" type="date" alt="Select the date the Finding will be remediated"/><br/>
+                            
+                            <label style="display: block">Planned remediation version</label>
+                            <input id="id_bulk_planned_remediation_version" name="planned_remediation_version" style="font-size: 100%;/* border-color: #ccc; *//* border-width: 1px; */border: 1px solid #ccc;"/><br/>
```
[the remark this answers is not in the document]
Comment (ptrovatelli) [reply dojo/templates/dojo/findings_list_snippet.html:99]: fixed

File: dojo/filters.py [hunk 2]
```diff
@@ -1221,6 +1221,7 @@ class FindingFilter(FindingFilterWithTags):
     mitigated = DateRangeFilter(label="Mitigated Date")
 
     planned_remediation_date = DateRangeOmniFilter()
+    planned_remediation_version = CharFilter(lookup_expr='icontains', label="Planned remediation version")
```
Comment (shipko) [inline dojo/filters.py:1224]: Please, add localization

File: dojo/forms.py [hunk 3]
```diff
@@ -940,6 +940,12 @@ class AddFindingForm(forms.ModelForm):
     references = forms.CharField(widget=forms.Textarea, required=False)
     publish_date = forms.DateField(widget=forms.TextInput(attrs={'class': 'datepicker', 'autocomplete': 'off'}), required=False)
     planned_remediation_date = forms.DateField(widget=forms.TextInput(attrs={'class': 'datepicker', 'autocomplete': 'off'}), required=False)
+    planned_remediation_version = forms.CharField(max_length=99, required=False)
+    effort_for_fixing = forms.ChoiceField(
+        required=False,
+        choices=EFFORT_FOR_FIXING_CHOICES,
+        error_messages={
+            'invalid_choice': 'Select valid choice: Low,Medium,High'})
```
Comment (shipko) [inline dojo/forms.py:948]: Please, add localization

File: dojo/forms.py [hunk 4]
```diff
@@ -1015,6 +1021,12 @@ class AdHocFindingForm(forms.ModelForm):
...
```
Comment (shipko) [inline dojo/forms.py:1029]: Please, add localization

File: dojo/templates/dojo/findings_list_snippet.html [hunk 5]
```diff
@@ -94,6 +94,10 @@ <h3 class="has-filters">
                                 <input id="id_bulk_date" name="date" type="date" alt="Select the date the Findings were discovered"/><br/>
                             <label style="display: block" for="id_bulk_planned_remediation_date">Planned Remediation Date</label>
                                 <input id="id_bulk_planned_remediation_date" name="planned_remediation_date" type="date" alt="Select the date the Finding will be remediated"/><br/>
+                            
+                            <label style="display: block" for="id_bulk_planned_remediation_version">Planned remediation version</label>
```
Comment (shipko) [inline dojo/templates/dojo/findings_list_snippet.html:98]: Please, add localization

File: dojo/templates/dojo/view_finding.html [hunk 6]
```diff
@@ -252,6 +252,9 @@ <h3 class="pull-left finding-title">
                     {% if finding.planned_remediation_date %}
                         <th>Planned Remediation</th>
                     {% endif%}
+                    {% if finding.planned_remediation_version %}
+                        <th>Planned Remediation version</th>
```
Comment (shipko) [inline dojo/templates/dojo/view_finding.html:256]: Please, add localization

File: dojo/templates/dojo/view_finding.html [hunk 7]
```diff
@@ -526,6 +532,9 @@ <h3 class="pull-left finding-title">
                 {% if 'is_finding_groups_enabled'|system_setting_enabled and finding.finding_group %}
                 <th>Group</th>
                 {% endif %}
+                {% if finding.effort_for_fixing %}
+                <th>Effort for fixing</th>
```
Comment (shipko) [inline dojo/templates/dojo/view_finding.html:536]: Please, add localization

**Comment by ptrovatelli** [comment]
@shipko tests finally went through after many reruns :) . All good to go? thanks

**Comment by shipko** [comment]
Good job!

**Comment by Maffooch** [comment]
@ptrovatelli I am a little nervous to attempt to fix these conflicts. They are pretty large chunks 😬

**Comment by ptrovatelli** [comment]
@Maffooch all good now :)

**Review by grendel513 — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**